src/c/config.cpp

```diff
@@ -181,4 +181,37 @@ void VBK_SetPopPayoutDelay(Config_t* params, int32_t val) {
 
 uint32_t VBK_GetMaxPopDataSize(Config_t* params) {
   return params->config->alt->mMaxPopDataSize;
+}
+std::vector<uint32_t> VBK_GetForkResolutionLookupTable(Config_t* params) {
+  return params->config->alt->mForkResolutionLookUpTable;
+}
+size_t VBK_GetMaxVbkBlocksInAltBlock(Config_t* params) {
+  return params->config->alt->mMaxVbkBlocksInAltBlock;
+}
+size_t VBK_GetMaxVTBsInAltBlock(Config_t* params) {
+  return params->config->alt->mMaxVTBsInAltBlock;
+}
+size_t VBK_GetMaxATVsInAltBlock(Config_t* params) {
+  return params->config->alt->mMaxATVsInAltBlock;
+}
+int32_t VBK_GetEndorsementSettlementInterval(Config_t* params) {
+  return params->config->alt->mEndorsementSettlementInterval;
+}
+uint32_t VBK_GetFinalityDelay(Config_t* params) {
+  return params->config->alt->mFinalityDelay;
+}
+uint32_t VBK_GetKeystoneInterval(Config_t* params) {
+  return params->config->alt->mKeystoneInterval;
+}
+uint32_t VBK_GetMaxAltchainFutureBlockTime(Config_t* params) {
+  return params->config->alt->mMaxAltchainFutureBlockTime;
+}
+altintegration::AltBlock VBK_GetBoostrapBlock(Config_t* params) {
+  return params->config->alt->getBootstrapBlock();
+}
```

Comment:
this one is not needed. VBK_GetBootstrapBlock is an extern - this function is implemented in Go